upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -305,6 +305,120 @@ There you can find links to upgrade notes for other versions too.
           
  - There is a new base html layout with horizontal menu and product filter placed in left panel, for detail information see [the separate article](upgrade-instructions-for-base-layout.md)
  
+- allow getting data for FE API from Elastic ([#1557](https://github.com/shopsys/shopsys/pull/1557))
+    - add and change fields in your elasticsearch definition files
+        ```diff
+        -   main_variant
+        +   is_main_variant
+         
+        +   "uuid": {
+        +      "type": "text"
+        +   },
+        +   "unit": {
+        +      "type": "text"
+        +   },
+        +   "is_using_stock": {
+        +      "type": "boolean"
+        +   },
+        +   "stock_quantity": {
+        +      "type": "boolean"
+        +   },
+        +   "variants": {
+        +       "type": "integer"
+        +   },
+        +   "main_variant": {
+        +      "type": "integer"
+        +   }
+        ```
+    !!! danger
```

Comment:
Admonitions (!!! danger) cannot be used in upgrade notes as they're meant to be read on Github and GitHub Markdown don't render them properly

Reply:
I removed it.